README.md

```diff
@@ -117,21 +96,22 @@ Spanner instance.
 
 Install Go ([download](https://golang.org/doc/install)) on your development
 machine if it is not already installed, configure the GOPATH environment
-variable if it is not already configured using 
-`export GOPATH=$(go env GOPATH)`, and 
+variable if it is not already configured, and
 [test your installation](https://golang.org/doc/install#testing).
 
 ### Installing HarbourBridge
 
 You can make a copy of the HarbourBridge codebase from the github repository
-and use "go run":
+and use "go run".
 
 ```sh
 git clone https://github.com/cloudspannerecosystem/harbourbridge
 cd harbourbridge
-go run github.com/cloudspannerecosystem/harbourbridge -driver=pg_dump < my_pg_dump_file
+go run github.com/cloudspannerecosystem/harbourbridge schema -source=postgres < my_pg_dump_file
```

Comment:
How about making this something that a first-time user can cut-and-paste and it'll just work?
In which case, maybe change the go run command to be:

go run github.com/cloudspannerecosystem/harbourbridge help

I know this is not very useful, but I think having something users can simply cut-and-paste for installation instructions is a big plus.

Reply:
Done, it's actually preferable to have `help` subcommand since that's more descriptive than the output of `commands` subcommand. Makes sense to do this, since right after this we do have more concrete examples of using `schema-and-data` subcommand.